ui/packages/app/src/project_setting/project_info/ProjectInfoForm.js

```diff
@@ -93,6 +93,7 @@ const ProjectInfoForm = ({ originalProject, fetchUpdates }) => {
               setIsValidLabels={setIsValidLabels}
               isValidLabels={isValidLabels}
               isDisabled={isDisabled}
+              labelsBlacklist={true}
```

Comment:
Hmm do we need this prop if this is always true? In other words, can we do away with this variable?